core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/DetailedStackStatus.java

```diff
@@ -13,8 +13,10 @@
     METADATA_COLLECTION(Status.CREATE_IN_PROGRESS),
     LOAD_BALANCER_METADATA_COLLECTION(Status.CREATE_IN_PROGRESS),
     TLS_SETUP(Status.CREATE_IN_PROGRESS),
-    PROVISIONED(Status.AVAILABLE),
+    PROVISIONED(Status.CREATE_IN_PROGRESS),
     PROVISION_FAILED(Status.CREATE_FAILED),
+    CLUSTER_RESET_FAILED(Status.CREATE_FAILED),
```

Comment:
Please make sure these changes are backward compatible in case we need a rollback.

Reply:
The `DetailedStackStatusConverter` would handle this.